ssl/ssl_cert.c

```diff
@@ -1034,7 +1036,7 @@ static int ssl_security_default_callback(const SSL *s, const SSL_CTX *ctx,
                 return 0;
             /* Level 3: forward secure ciphersuites only */
```

Comment:
Nit: I think it might be more readable to have L1023 initialize `pfs_mask` as zero, and set it to `SSL_kDHE | SSL_kECDHE | SSL_kDHEPSK | SSL_kECDHEPSK` here, so one can immediately verify what the check is doing.